rest_api/controller/pipeline.py

```diff
@@ -0,0 +1,54 @@
+# coding: utf8
+import shutil
+import uuid
+import os
+from pathlib import Path
+from fastapi import APIRouter, UploadFile, File, HTTPException
+from typing import List
+
+from rest_api.config import PIPELINES_DIR
+from rest_api.controller.utils import PipelineHelper, PipelineSchema
+
+router = APIRouter()
+pipeline_helper = PipelineHelper(PIPELINES_DIR)
+
+
+@router.get("/pipelines", response_model=List[PipelineSchema], response_model_include={'name', 'type', 'status'})
```

Comment:
We could make this endpoint rather "high-level" to allow clients to get an overview of the pipelines. For the `PipelineSchema`, there's already a dedicated endpoint. 

We can start with a simple dict like: `{"name": "MyPipeline", "type": "query", "status": "active"}`

Reply:
@oryx1729 I am already returning the same output. somehow it is showing all output in API view.

Reply:
I am already returning the same output.